packages/iTwinUI-react/src/core/Table/columns/actionColumn.tsx

```diff
@@ -99,6 +101,8 @@ export const ActionColumn = <T extends Record<string, unknown>>({
           menuItems={headerCheckBoxes}
           onHide={() => setIsOpen(false)}
           onShow={() => setIsOpen(true)}
+          className={className}
+          style={{ maxHeight: '300px', overflow: 'scroll', ...style }} // Default style (max height) that can be overridden/unset
```

Comment:
While it's nice that the user can override `style`, this isn't the best way to do it. We should try to add scrolling through CSS. We already have a class for this purpose.

```jsx
className='iui-scroll'
```

And we can also merge the user's name using `import cx from 'classnames'`:
```suggestion
          className={cx('iui-scroll', className)}
```

Reply:
Thanks for the idea! I will not add the `style` parameter to the `ActionColumn`, but should I at least have a style with `max-height: 300px`? Because without it, the dropdown menu always expands to fit and so there will not be any scrolling

Reply:
That's interesting. I thought our `.iui-scroll` class includes the max-height, but apparently not. I feel like it _should_ though. Maybe lets add it in `itwinui-css`?

@bentleyvk What do you think?

Reply:
In the interest of not blocking this PR:
- Lets add an inline style that sets max-height to `315` (same as our other dropdown components). 
- Lets open an issue in the `iTwinUI` repo to handle max-height there.

Reply:
Thanks, I created an [issue in the iTwinUI repo](https://github.com/iTwin/iTwinUI/issues/733) and added an inline style `maxHeight: 315px` to the `DropdownMenu` in `ActionColumn`. I will edit the new tests to work with this new props structure

Reply:
Make sure to merge user's `style` prop (from `columnManager.dropdownProps`) into it

Reply:
**[DONE]** Thanks for reminding